cli/cmd/volume.go

```diff
@@ -87,17 +88,27 @@ var volumeStartCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		validateNArgs(cmd, 1, 1)
 		volname := cmd.Flags().Args()[0]
-		fmt.Println("START:", volname)
+		err := client.VolumeStart(volname)
+		if err != nil {
+			log.WithField("volume", volname).Println("volume start failed")
+			failure(fmt.Sprintf("volume start failed with: %s", err.Error()), 1)
+		}
+		fmt.Printf("Volume %s started successfully\n", volname)
 	},
 }
 
 var volumeStopCmd = &cobra.Command{
-	Use:   "stop",
+	Use:   "stop [flags] <VOLNAME>",
```

Comment:
What are the 'flags' here? Do you even need to mention it here? I thought cobra automatically adds all defined flags to the commands help outpu.

Reply:
I just verified about [flags], it is optional. cobra will add it if not added. Adds it to the end.

Sorry @nnDarshan, I requested you to add flag in last patchset. It is optional.

cobra doesn't have any facility for positional arguments as for as I know. So positional args required.